apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -845,6 +995,290 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
             "console:manage.features.user.modals.bulkImportUserWizard.wizardSummary.tableStatus.failed" );
     };
 
+    /**
+     * Render Multiple Users mode selection section.
+     */
+    const resolveMultipleUsersModeSelection = (): ReactElement => {
+        return(
+            <Grid.Row>
+                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                    <Button.Group
+                        size="large"
+                        labeled
+                        basic
+                    >
+                        {
+                            Object.values(MultipleInvites).map((mode: string, index: number) => {
+                                return(
+                                    <>
+                                        <Popup
```

Comment:
Don't we need to pass `key` prop?

Reply:
There is no requirement to pass the key. WDYT?